resources/metadata/hadolint.yaml

```diff
@@ -0,0 +1,76 @@
+metadata:
+  name: hadolintExecute
+  description: Executes the Haskell Dockerfile Linter which is a smarter Dockerfile linter that helps you build [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) Docker images.
+  longDescription: |-
+    Executes the Haskell Dockerfile Linter which is a smarter Dockerfile linter that helps you build [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) Docker images.
+    The linter is parsing the Dockerfile into an abstract syntax tree (AST) and performs rules on top of the AST.
+spec:
+  inputs:
+    secrets:
+      - name: configurationCredentialsId
+        type: jenkins
+        description: Jenkins 'Username with password' credentials ID containing username/password for access to your remote configuration file.
+    params:
+      - name: configurationUrl
+        type: string
+        description: URL pointing to the .hadolint.yaml exclude configuration to be used for linting. Also have a look at `configurationFile` which could avoid central configuration download in case the file is part of your repository.
+        mandatory: false
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        default: null
+      - name: configurationUsername
+        type: string
+        description: The username to authenticate
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        secret: true
+        resourceRef:
+          - name: configurationCredentialsId
+            type: secret
+            param: username
+      - name: configurationPassword
+        type: string
+        description: The password to authenticate
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        secret: true
+        resourceRef:
+          - name: configurationCredentialsId
+            type: secret
+            param: password
+      - name: dockerFile
```

Comment:
uppercase `F`? Shouldn't this rather be `dockerfile`? If compatibility issue maybe use `dockerFile` as alias?

Reply:
One doubt, is aliasing for step parameter supported or only for the config file?

Reply:
@OliverNocon Aliasing is not supported for `--stepConfigJSON`, so this would be a breaking change. Any ideas?

Reply:
reverted the change

Reply:
added `dockerfile` as alias